### ISSUE TITLE
Extension doesn't support 64 characters(mini BLE)

### ISSUE THREAD

**Issue by ThunderKing54** [mooltipass/extension#83]
8<------------------------ BUG REPORT -----------------------------------------
### Expected behavior
The extension should accept 64 characters for the Mooltipass mini BLE
### Actual behavior
The extension doesn't accept more than 31 characters for the Mooltipass mini BLE. A message appears that only 31 characters are supported.
### Step by step guide to reproduce the problem
1. go to any website and login with credentials bigger then 31 characters
2. a message appears that only 31 characters are supported.

### ExtensionVersion
1.8.0

### Operating System
Windows 10 Pro Build 18362

### Browser
Firefox 70.0.1

### Mooltipass Device
- The Mooltipass Mini BLE

8<------------------------ END BUG REPORT -------------------------------------

**Comment by ThunderKing54** [comment]
This problem is still there. In Moolticute I'm able to add 64 character passwords

**Comment by limpkin** [comment]
indeed it is! at the moment we don't have anyone working on the extension... that should hopefully change soon.

**Comment by Yzord** [comment]
When is soon as this is still current and i have a lot of passwords which are using more then 31 characters

**Comment by limpkin** [comment]
@Yzord did you confirm this issue still exists? it should be fixed on moolticute side of things, allowing you to import you credentials.

**Comment by Yzord** [comment]
> @Yzord did you confirm this issue still exists? it should be fixed on moolticute side of things, allowing you to import you credentials.

Yes, i have multiple passwords which are not accepted. It gave me a notification that the password is too long and that Mooltipass is not supporting it.

<img width="805" alt="Screenshot 2021-07-01 at 09 42 18" src="https://user-images.githubusercontent.com/5834848/124085774-ae36d500-da50-11eb-9bda-972bd8ff3502.png">

**Comment by spoelstraethan** [comment]
Funny enough I actually get this warning after having the MiniBLE manually type in a password longer than 31 characters, but since the login credentials are associated with a few domains the extension isn't aware that I already have it saved.

**Comment by limpkin** [comment]
is this still happening?
